libs/components/lookup/src/lib/modules/lookup/types/lookup-show-more-native-picker-config.ts

```diff
@@ -16,6 +16,14 @@ export interface SkyLookupShowMoreNativePickerConfig {
   /**
    * The title for the picker.
    * @default "Select an option/Select options"
+   * @deprecated Use the `selectionDescriptor` input to give context to the title and accessibility labels instead.
    */
   title?: string;
+
+  /**
+   * A descriptor for the item or items being selected. Use a plural term when the lookup's `selectMode` is set to `multiple`; otherwise, use a singular term. The descriptor helps set the picker's `aria-label` attributes for the multiselect toolbar controls, the search input, and the save button to provide text equivalents for screen readers [to support accessibility](https://developer.blackbaud.com/skyux/components/checkbox#accessibility).
+   * For example, when the descriptor is “constituents,” the search input’s `aria-label` is “Search constituents.” For more information about the `aria-label` attribute, see the [WAI-ARIA definition](https://www.w3.org/TR/wai-aria/#aria-label).
```

Comment:
It looks like there's a mix of characters for quotation marks and apostrophes. We should make them consistent. `"` versus `“”` and `'` versus `’`.

Reply:
I'm not sure how that happened..... but it was that way in all the recent similar descriptions. I have updated all of them in this PR to use the standard quotes/apostrophes.